### DISCUSSION TITLE
Should we reuse the built-in postgres server, or host our own?

### DISCUSSION THREAD

**Discussion by Yattabyte** [apache/superset#13175] [Q&A / Help] [answered]
I can easily set up additional databases via postgres on my local network, but superset appears to also have postgres built into its docker image, though I can't seem to connect to it with pgadmin.

Is the built-in postgres server meant to be reused by us, our ignored and coupled with whatever servers we host?
(Using Docker-compose)

If so, what is the credentials to access it? I thought localhost @ 5432 admin/admin would suffice.
If not, can we permanently purge the example data so it doesn't pollute the interface every time we restart the server?

**Comment by willbarrett** [comment, marked as answer]
Hi @Yattabyte - the distributed `docker-compose` setup is intended for development use only. I would recommend using the official Docker image for Superset in conjunction with your own hosted postgres server if you are attempting a production deployment of Superset.

**Reply by Yattabyte** [reply]
@willbarrett Thank you so much for clarifying that 👍
